client/client-multi/client-c/src/kaa_user.c

```diff
@@ -32,14 +34,19 @@
 
 #define EXTERNAL_SYSTEM_AUTH_FIELD      0x00
 
+
+
 extern kaa_transport_channel_interface_t *kaa_channel_manager_get_transport_channel(kaa_channel_manager_t *self
                                                                                   , kaa_service_t service_type);
 
 typedef struct {
-    size_t  user_external_id_len;
-    size_t  user_access_token_len;
-    char   *user_external_id;
-    char   *user_access_token;
+    char          *user_external_id;
+    size_t        user_external_id_len;
+    char          *user_access_token;
+    size_t        user_access_token_len;
+    char          *user_verifier_token;
+    size_t        user_verifier_token_len;
+    destroy_fn    destroy;
```

Comment:
To tell the truth, there is no any specific reason. It is a mini enterprise :)
I will get rid off this callback and replace it with direct call of the destroy function.